.circleci/config.yml

```diff
@@ -142,6 +142,9 @@ jobs:
       - install_chrome
       - attach_workspace:
           at: /home/circleci/
+      - run:
+          name: Setup python by installing protobuf and enum
+          command: pip install enum34==1.1.10 protobuf==3.13.0
```

Comment:
Can these steps be run in the setup job around line 115? With the version numbers duplicated throughout the file, it's going to be all to easy to forget to update one of the occurrences if we ever upgrade these packages

Reply:
Answered above.